### PR TITLE
AnP: Add status message to release command

### PR DESCRIPTION
**Pull request by ebelter**

### CODE REVIEW

File: lib/perl/Genome/Config/AnalysisProject/Command/Release.t [hunk 1]
```diff
@@ -17,14 +17,12 @@ my $ap = Genome::Config::AnalysisProject->create(
...
 
-is($ap->status, 'Hold', 'initial status should be pending');
-
-my $cmd = $class->create(analysis_projects => [$ap]);
-$cmd->execute();
-
-is($ap->status, 'In Progress', 'it should set the status to in progress');
+my $cmd = $class->execute(analysis_projects => [$ap]);
+ok($cmd->result, 'execute cmd');
+my $expexted_status = 'In Progress';
```
Comment (tmooney) [inline lib/perl/Genome/Config/AnalysisProject/Command/Release.t]: `expected` has traded its `c` for a surplus `x`.

File: lib/perl/Genome/Config/AnalysisProject/Command/Release.t [hunk 2]
```diff
@@ -17,14 +17,12 @@ my $ap = Genome::Config::AnalysisProject->create(
     name => 'Test Project',
     status => 'Hold',
 );
+is($ap->status, 'Hold', "AnP status is'Hold'");
```
Comment (tmooney) [inline lib/perl/Genome/Config/AnalysisProject/Command/Release.t]: It'd be nice to have a space between `is` and `'Hold'`.

**Comment by tmooney** [comment]
+1 :spaghetti:

**Comment by brummett** [comment]
+1

**Pull request merged**